crates/bevy_ecs/src/system/commands/parallel_scope.rs

```diff
@@ -64,16 +62,8 @@ impl<'w, 's> ParallelCommands<'w, 's> {
     ///
     /// For an example, see the type-level documentation for [`ParallelCommands`].
     pub fn command_scope<R>(&self, f: impl FnOnce(Commands) -> R) -> R {
-        let store = &self.state.thread_local_storage;
-        let command_queue_cell = store.get_or_default();
-        let mut command_queue = command_queue_cell.take();
-
-        let r = f(Commands::new_from_entities(
-            &mut command_queue,
-            self.entities,
-        ));
-
-        command_queue_cell.set(command_queue);
-        r
+        self.state
+            .thread_queues
+            .scope(|command_queue| f(Commands::new_from_entities(command_queue, self.entities)))
```

Comment:
```suggestion
            .scope(|queue| {
                 let commands = (Commands::new_from_entities(queue, self.entities));
                 f(commands)
             })
```

Purely a stylistic preference. Feel free to ignore.